apps/game_backend/lib/game_backend/units/skills/effects/target_strategy.ex

```diff
@@ -22,11 +27,13 @@ defmodule GameBackend.Units.Skills.Effects.TargetStrategy do
 
   def dump(strategy), do: {:ok, target_strategy_to_string(strategy)}
 
+  defp target_strategy_to_string("all"), do: "random"
```

Comment:
Rather than storing it wrongly on the DB, we should keep this as it was and change `create_effect_map` in `Simulator` to always set target_strategy to "random".

Same design decision should be followed by other fallbacks